stdlib/math.gr

```diff
@@ -0,0 +1,16 @@
+export *
+
+import foreign wasm mathSqrt : (Number) -> Number as sqrt from 'stdlib-external/math'
+import foreign wasm mathCeil : (Number) -> Number as ceil from 'stdlib-external/math'
+import foreign wasm mathFloor : (Number) -> Number as floor from 'stdlib-external/math'
+import foreign wasm mathTrunc : (Number) -> Number as trunc from 'stdlib-external/math'
+import foreign wasm mathAbs : (Number) -> Number as abs from 'stdlib-external/math'
+import foreign wasm mathMin : (Number, Number) -> Number as min from 'stdlib-external/math'
+import foreign wasm mathMax : (Number, Number) -> Number as max from 'stdlib-external/math'
+import foreign wasm mathNeg : (Number) -> Number as neg from 'stdlib-external/math'
+import foreign wasm mathRound : (Number) -> Number as round from 'stdlib-external/math'
```

Comment:
I believe that while you were still working on this, we had a breaking change for quotes:
```suggestion
import foreign wasm mathSqrt : (Number) -> Number as sqrt from "stdlib-external/math"
import foreign wasm mathCeil : (Number) -> Number as ceil from "stdlib-external/math"
import foreign wasm mathFloor : (Number) -> Number as floor from "stdlib-external/math"
import foreign wasm mathTrunc : (Number) -> Number as trunc from "stdlib-external/math"
import foreign wasm mathAbs : (Number) -> Number as abs from "stdlib-external/math"
import foreign wasm mathMin : (Number, Number) -> Number as min from "stdlib-external/math"
import foreign wasm mathMax : (Number, Number) -> Number as max from "stdlib-external/math"
import foreign wasm mathNeg : (Number) -> Number as neg from "stdlib-external/math"
import foreign wasm mathRound : (Number) -> Number as round from "stdlib-external/math"
```